xlgui/guiutil.py

```diff
@@ -730,4 +732,39 @@ def without_model(tv):
             tv.scroll_to_cell(scroll_to, None, True, 0, 0)
 
 
+def get_builder(uifile: str) -> Gtk.Builder:
+    """
+    Loads Gtk.Builder with uifile.
+    In case of platform win32 each translatable node in the ui is translated here
```

Comment:
This is due to the bug you mentioned (https://gitlab.gnome.org/GNOME/pygobject/-/issues/422), right? Could you add the link, so we remember why this hack is needed?

Reply:
Added